vizro-core/src/vizro/tables/dash_aggrid.py

```diff
@@ -0,0 +1,78 @@
+"""Module containing the standard implementation of `dash-ag-grid.AgGrid`."""
+
+import dash_ag_grid as dag
+import pandas as pd
+
+from vizro.models.types import capture
+from vizro.tables._utils import _set_defaults_nested
+
+FORMAT_CURRENCY_EU = """d3.formatLocale({
+  "decimal": ",",
+  "thousands": "\u00a0",
+  "grouping": [3],
+  "currency": ["", "\u00a0€"],
+  "percent": "\u202f%",
+  "nan": ""
+})"""
+
+DATA_TYPE_DEFINITIONS = {
+    "number": {
+        "baseDataType": "number",
+        "extendsDataType": "number",
+        "columnTypes": ["numberColumn", "rightAligned"],
+        "appendColumnTypes": True,
+        "valueFormatter": {"function": "params.value == null ? 'NaN' : String(params.value)"},
+    },
+    "dollar": {
+        "baseDataType": "number",
+        "extendsDataType": "number",
+        "valueFormatter": {"function": "d3.format('($,.2f')(params.value)"},
+    },
+    "euro": {
+        "baseDataType": "number",
+        "extendsDataType": "number",
+        "valueFormatter": {"function": f"{FORMAT_CURRENCY_EU}.format('$,.2f')(params.value)"},
+    },
+    "percent": {
+        "baseDataType": "number",
+        "extendsDataType": "number",
+        "valueFormatter": {"function": "d3.format(',.1%')(params.value)"},
+    },
+    "numeric": {
+        "baseDataType": "number",
+        "extendsDataType": "number",
+        "valueFormatter": {"function": "d3.format(',.1f')(params.value)"},
+    },
+}
+
+
+@capture("aggrid")
+def dash_ag_grid(data_frame=None, **kwargs):
```

Comment:
```suggestion
def dash_ag_grid(data_frame, **kwargs):
```

Unless I missed something, this is not optional.